mozilla_django_oidc/auth.py

```diff
@@ -68,12 +101,28 @@ def authenticate(self, code=None, state=None):
             user_response = requests.get('{url}?{query}'.format(url=self.OIDC_OP_USER_ENDPOINT,
                                                                 query=query))
             user_info = user_response.json()
+            email = user_info.get('email')
+            if email is None:
```

Comment:
I would prefer ``if not email`` in case for some reason ``email`` is in dict but is empty.